packages/spec/helpers/test-helpers.js

```diff
@@ -29,6 +31,7 @@ const startServer = ({ cwd, command, argv = [] }) =>
     const args = [hopsBin, command].concat(argv);
     debug('Spawning server', [hopsBin, ...args].join(' '));
     const started = spawn(process.argv[0], args, {
+      env,
```

Comment:
Why do we need this? The child process has access to the same environment as the parent process. We would only need to set this if we want to change the environment.

Reply:
Setting `process.env.WHATEVER` before calling `HopsCLI.start(…)` didn't lead to the env var being present in the spawned process. Maybe it works, if the env var is passed into the initiating process (rather than fiddling with the `process.env`-object), but that wouldn't really work, because we can only specifiy env vars for the whole test suite if I'm not mistaken.